### PR TITLE
Update setting_user_ids.md

### PR DESCRIPTION
**Pull request by chshapiro**
https://jira.braze.com/browse/SDK-753

### CODE REVIEW

**Review by KellieHawks — APPROVED** [review]
lgtm!

**Review by tbelote — APPROVED** [review]
Lgtm

File: _docs/_developer_guide/platform_integration_guides/ios/analytics/setting_user_ids.md [hunk 1]
```diff
@@ -57,6 +57,10 @@ Appboy.sharedInstance()?.changeUser("YOUR_USER_ID")
 {% endtab %}
 {% endtabs %}
 
+{% alert warning %}
+Be sure to call this method in your application's main thread. Calling the method asynchronously can lead to broken functionality.
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/ios/analytics/setting_user_ids.md:61]: broken functionality -> undefined behavior

**Review by Bucimis — APPROVED** [review]

**Review by KellieHawks — APPROVED** [review]
lgtm!

**Pull request merged**